data/json/items/tool/smoking.json

```diff
@@ -58,6 +58,36 @@
     "qualities": [ [ "SMOKE_PIPE", 1 ] ],
     "melee_damage": { "bash": 1 }
   },
+  {
+    "id": "pipe_water",
+    "type": "TOOL",
+    "name": { "str": "water pipe" },
+    "looks_like": "bottle_plastic",
+    "description": "A mass-manufacured water-pipe, otherwise known as a 'bong.' Headshops often insist these are used solely for tobacco, despite their iconic status as cannabis paraphernalia.",
```

Comment:
```suggestion
    "description": "A mass-manufactured water-pipe, otherwise known as a 'bong.'  Head shops often insist these are used solely for tobacco, despite their iconic status as cannabis paraphernalia.",
```